pkgs/by-name/mi/mingtest/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  stdenv,
+  cmake,
+  texliveSmall,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  name = "mingtest";
+  version = "0.1.9-4";
+  src = fetchFromGitHub {
+    owner = "craflin";
+    repo = "mingtest";
+    rev = "b6af8ada0908ea855a563ef43d5543bab8512d7d";
```

Comment:
You should reference version as tag whenever possible, using `stdenv.mkDerivation (finalAttrs: { ... }` and `rev = "refs/tags/${finalAttrs.version}"`

Reply:
Something like this 

```
mkDerivation (finalAttrs: {
  version = "1.2.3";
  src = fetchFromGitHub {
    repo = finalAttrs.pname;
    rev = "v${finalAttrs.version}"; 
    hash = "..some literal hash..";
  };
  ... ;
})
```

Reply:
@jackyliu16 never use `finalAttrs.pname` in a url, see #326683